GUI/QtQuick/qml/Send/AdvancedDetails.qml

```diff
@@ -440,6 +440,14 @@ ColumnLayout  {
                     width: BSSizes.applyScale(504)
                     height: BSSizes.applyScale(70)
 
+                    function getMax() {
+                        if (!isRBF && !isCPFP && txInputsSelectedModel.rowCount > 1) {
+                            return (txInputsSelectedModel.sourceModel.balance - txOutputsModel.totalAmount).toFixed(8)
```

Comment:
This is not the correct max amount calculation, you miss the expected fee here. You need to set the correct params for tempRequest retrieval instead.